src/main/java/pl/connectis/projektgrupowy/domain/Book.java

```diff
@@ -7,12 +7,25 @@
 @Entity
 @Data
 public class Book {
-
+    
+    public Long getId() {
```

Comment:
Do you need getter if there is @Data annotation?

Reply:
removed